src/Symfony/Component/Process/CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 CHANGELOG
 =========
 
+6.2.0
+-----
+
+* added support for `Process::findExecutable()` to find executables independent of `open_basedir` configuration
```

Comment:
s/added/Add + missing space at start of line

(we changed our policy regarding formatting in changelog files. new entries should use the new style)

Reply:
Fixed.